nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -435,6 +435,17 @@ async def del_widgets(self, widget: str):
         entities_key = KB_WIDGETS_WIDGET.format(kbid=self.kbid, id=widget)
         await self.txn.delete(entities_key)
 
+    async def get_synonyms(self, synonyms: PBSynonyms):
+        pbsyn = await Synonyms(self.txn, self.kbid).get()
+        if pbsyn is not None:
+            synonyms.CopyFrom(pbsyn)
+
+    async def set_synonyms(self, synonyms: PBSynonyms):
+        await Synonyms(self.txn, self.kbid).set(synonyms)
+
+    async def delete_synonyms(self):
+        await Synonyms(self.txn, self.kbid).clear()
+
```

Comment:
If every call will create a `Synonyms` instance and call a method, why not let the clients use Synonyms directly as a part of the ORM?

If you find it useful, maybe a `get_synonyms_klass` method following the idea of other classes on the ORM.